test/linter/test-style.js

```diff
@@ -78,12 +78,12 @@ function processData(filename, logger) {
 
   if (expected !== expectedBrowserSorting) {
     hasErrors = true;
-    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedBrowserSorting)}}}\n{blue     Tip: Run {bold npm run fix} to fix sorting automatically}`);
+    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(actual, expectedBrowserSorting)}}}\n{blue     Tip: Run {bold npm run fix} to fix sorting automatically}`);
   }
 
   if (actual !== expectedFeatureSorting) {
```

Comment:
```suggestion
  if (expected !== expectedFeatureSorting) {
```

Should this be "expected"? I think otherwise you will also get feature sorting error messages when you have just indentation / stylistic errors. At least that's what I'm seeing.

Reply:
You've got a point, and the browser sorting has the same thing -- yeah, let's apply the change!

(I'm also wondering if we should rename the variable to `expectedBeforeSorting`?)